app/agent/model.scala

```diff
@@ -28,12 +29,30 @@ trait IndexedItemWithStack extends IndexedItem {
 
 case class AWSAccount(accountNumber: Option[String], accountName: String)
 
-abstract class CollectorSet[T](val resource:ResourceType, accounts: Accounts) extends Logging {
+sealed trait AwsRegionType
+case object Global extends AwsRegionType
+case object Regional extends AwsRegionType
+
+/** The awsRegionType specifies whether an AWS collector is global or regional. The AWS_GLOBAL region is required for
+ * global services, such as Route 53. */
```

Comment:
```suggestion
/** A CollectorSet knows how to create a set of collectors for a given resource type that typically
 *  spans multiple accounts, which can be of different underlying platforms. 
 *  A CollectorSet creates an appropriate set of Collector instances for each account and region.
 *
 * @param resourceType the name of the resource that this CollectorSet is responsible for
 * @param accounts the set of accounts to collect this resource from
 * @param awsRegionType some resourceTypes in AWS have a single Global instance instead of Regional 
 * instances. If a CollectorSet processes `AmazonOrigin` origins then you should specify whether the AWS 
 * collector is global (such as Route53) or regional (such as EC2 instances).
 * @tparam T the class that represents a collected instance of the resource
 */
```